app/javascript/packages/document-capture/components/unknown-error.tsx

```diff
@@ -13,6 +14,21 @@ interface UnknownErrorProps extends ComponentProps<'p'> {
   hasDismissed: boolean;
 }
 
+function formattedIdTypeMsg({ altFailedDocTypeMsg, acceptedIdUrl }) {
+  return formatHTML(altFailedDocTypeMsg, {
+    a: ({ children }) => (
+      <a
+        target="_blank"
+        className="usa-link--external"
+        rel="noopener noreferrer"
+        href={acceptedIdUrl}
+      >
+        {children}
+      </a>
```

Comment:
I think the pattern we use for this is the `Link` component with `isExternal: true`
```javascript
// document-capture/components/in-person-prepare-step.tsx
<Link href={securityAndPrivacyHowItWorksURL}>
  {t('in_person_proofing.body.prepare.privacy_disclaimer_link')}
</Link>

// components/document-capture-troubleshooting-options.tsx
url: getFailureToProofURL(location),
text: t('idv.troubleshooting.options.get_help_at_sp', { sp_name: spName }),
isExternal: true,
```

Reply:
@charleyf , thank you and good to know.

Reply:
fixed.